src/main/java/callhub/connect/use_case/SessionController.java

```diff
@@ -54,6 +51,18 @@ public ResponseEntity<String> joinSession(@PathVariable String code) {
         return new ResponseEntity<>(gson.toJson(responseBody), headers, HttpStatus.OK);
     }
 
+    @GetMapping("/transcript/{code}")
+    public ResponseEntity<String> getTranscript(@PathVariable String code) {
+        HttpHeaders headers = new HttpHeaders();
+        Session session = sessionRepository.getSessionsByActiveAndCode(true, code);
```

Comment:
Here you check if the session exists, please add an exception in case an invalid session code was added or an error occurs. See https://github.com/Callhub-Connect/callhub-backend/blob/pdf-session-connection/src/main/java/callhub/connect/controllers/FileController.java  if you want an example (at the end of the file)